src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -199,8 +191,30 @@ public void start(Stage primaryStage) throws Exception {
         thestage.show();
     }
 
-    private void buttonClicked(ActionEvent e)
-    {
+    private void buttonClicked(ActionEvent e) {
         thestage.setScene(mainScene);           // TODO: Call a controller function here instead
     }
+
+    private void addConversation(ActionEvent e) {
```

Comment:
Some comments on these functions explaining exactly what they do would make it easier to understand.